lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_build_prechecks.rb

```diff
@@ -9,16 +9,15 @@ def self.run(params)
 
         Fastlane::Helper::GitHelper.ensure_on_branch!('release') unless other_action.is_ci()
 
-        app = params[:app]
         message = ''
-        beta_version = Fastlane::Helper::Android::VersionHelper.get_release_version(product_name: app) unless !params[:beta] && !params[:final]
-        alpha_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version(app) if params[:alpha]
+        beta_version = Fastlane::Helper::Android::VersionHelper.get_release_version() unless !params[:beta] && !params[:final]
+        alpha_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version() if params[:alpha]
```

Comment:
Nitpick: Manual Rubocop 💅

```suggestion
        beta_version = Fastlane::Helper::Android::VersionHelper.get_release_version unless !params[:beta] && !params[:final]
        alpha_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version if params[:alpha]
```